packages/analytics-js-integrations/src/integrations/GA4/browser.js

```diff
@@ -27,8 +30,10 @@ export default class GA4 {
     this.debugView = config.debugView || false;
     this.capturePageView = config.capturePageView || 'rs';
     this.isHybridModeEnabled = config.connectionMode === 'hybrid';
+    this.piiPropertiesToIgnore = config.piiPropertiesToIgnore || [];
```

Comment:
Don't we want to fallback to some default PII fields?

Reply:
Lets keep empty for now don't wat to take chances on assuming wrong fields

Reply:
i guess as we are already giving option to choose, In future is some customer want to send default PII fields then they can't do it. batter to give whole control over fields to customer